compareAll.sh

```diff
@@ -117,9 +119,15 @@ function compenv() {
 
 function genTrainingDS() {
     ensuremldirs
-    for tag in ${tags[@]}; do
-        logandrun ./ml/create_training_dataset.sh ${erasarg} ${channelsarg} ${tag}
-    done
```

Comment:
`erasarg` should be `all` if you pass ``all`. I dont think this is needed

Reply:
[[ "" = $( echo ${eras} ) ]] && eras=("2016" "2017" "2018") erasarg="2016,2017,2018"

Ne, diese Zeile macht erasarg zu "2016,2017,2018" wenn man davor "all" hatte.

Reply:
ah, richtig :/